Lib/test/test_code.py

```diff
@@ -266,8 +266,8 @@ def func2():
             ("co_stacksize", 0),
             ("co_flags", code.co_flags | inspect.CO_COROUTINE),
             ("co_firstlineno", 100),
-            ("co_code", code2.co_code),
             ("co_consts", code2.co_consts),
+            ("co_code", code2.co_code),
```

Comment:
Why have you swapped these two lines?

Reply:
@markshannon Fixed it. I thought the changes to `code` were permanent and that would fix the error so I swapped those two. The change is outdated now though.